src/index.coffee

```diff
@@ -1,48 +1,52 @@
+require( 'es6-promise').polyfill()
 ZooUserStringGetter = require 'zooniverse-user-string-getter'
 
 module.exports = class GeordiClient
 
-  GEORDI_STAGING_SERVER_URL:
-    'https://geordi.staging.zooniverse.org/api/events/'
-  GEORDI_PRODUCTION_SERVER_URL:
-    'https://geordi.zooniverse.org/api/events/'
-
+  GEORDI_SERVER_URL:
+    staging: 'https://geordi.staging.zooniverse.org/api/events/'
+    production: 'https://geordi.zooniverse.org/api/events/'
+  
+  GEORDI_DATABASE_FIELDS: [
```

Comment:
I copied the array from this line here https://zooniverse.slack.com/archives/geordi-into-panoptes/p1465383282000008
Are there fields missing?